recipes/highfive/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "HighFive" %}
+{% set version = "2.0" %}
+{% set sha256 = "deee33d7f578e33dccb5d04771f4e01b89a980dd9a3ff449dd79156901ee8d25" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: HighFive-v{{ version }}.tar.gz
+  url: https://github.com/BlueBrain/HighFive/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - hdf5
+    - boost
+
+test:
+  commands:
+    # Testing for installation of headers - non-exhaustive list.
+    - test -d $PREFIX/include/highfive  # [unix]
+    - test -f $PREFIX/include/highfive/H5Attribute.hpp  # [unix]
+    - test -d $PREFIX/include/highfive/bits  # [unix]
+    - test -f $PREFIX/include/highfive/bits/H5File_misc.hpp  # [unix]
+    - test -f $PREFIX/include/highfive/H5Exception.hpp  # [unix]
+    - test -f $PREFIX/share/HighFive/CMake/HighFiveConfig.cmake  # [unix]
+    - test -f $PREFIX/share/HighFive/CMake/HighFiveTargets.cmake  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\highfive\H5Attribute.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\include\highfive\bits\H5File_misc.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\include\highfive\H5Exception.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\share\HighFive\CMake\HighFiveConfig.cmake (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\share\HighFive\CMake\HighFiveTargets.cmake (exit 0) else (exit 1)  # [win]
+
+about:
+  home: https://github.com/BlueBrain/HighFive
+  license: BSL-1.0
+  license_file: LICENSE
+  summary: 'Header-only C++ HDF5 interface'
+  description: 'HighFive is a modern C++/C++11 friendly interface for libhdf5.'
+  dev_url: https://github.com/BlueBrain/HighFive
+
+extra:
+  recipe-maintainers:
+    - wolfv
+    - SylvainCorlay
+    - JohanMabille
```

Comment:
@wolfv @JohanMabille - are you willing to be maintainers here?

Reply:
You can remove yourselves in the feedstock if you want :wink:

Reply:
@wolfv @JohanMabille no way :smirk: